src/index.test.ts

```diff
@@ -0,0 +1,139 @@
+import { mocked } from 'ts-jest/utils';
+
+import { readableFromArray } from './test-helpers/stream-utils';
+import * as express from 'express';
+import * as request from 'supertest';
+
+import * as mockDataset from './test-helpers/mock-dataset.json';
+
+describe('Output Plugin', () => {
+  let mockConfigModule;
+  let plugin;
+  let app: express.Application;
+
+  const siteHostName = 'download-test-qa-pre-a-hub.hubqa.arcgis.com';
+
+  function buildPluginAndApp () {
+    const Output = require('./');
+
+    const plugin = new Output();
+    plugin.model = {
+      pullStream: jest.fn().mockResolvedValue(readableFromArray([mockDataset])),
+    };
+
+    app = express();
+    app.get('/dcat', plugin.serve.bind(plugin));
+
+    return [ plugin, app ];
+  }
+
+  beforeEach(() => {
+    jest.resetModules();
+
+    // const {
```

Comment:
Do we still need this commented out code?

Reply:
It's coming back in the next PR.